reconfiguration/include/reconfiguration/pruning_utils.hpp

```diff
@@ -0,0 +1,133 @@
+// Concord
+//
+// Copyright (c) 2018-2021 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include "concord.cmf.hpp"
+#include <boost/endian/buffers.hpp>
+#include <string>
+#include <type_traits>
+#include <unordered_set>
+#include <bftengine/IStateTransfer.hpp>
+#include <db_interfaces.h>
+#include <future>
+#include "Crypto.hpp"
+#include "bftengine/ControlStateManager.hpp"
+
+namespace concord::reconfiguration::pruning {
+class PruningException : public std::exception {
```

Comment:
If there's no special treatment for the pruning exceptions, we don't need them.
Just throw std::runtime_error